ci/conda-pack.sh

```diff
@@ -1,7 +1,15 @@
 #!/bin/bash
 set -e
 
-CONDA_ENV_NAME="rapids${RAPIDS_VER}_cuda${CUDA_VER}_py${PYTHON_VER}"
+RAPIDS_VER="23.06a"
```

Comment:
I think this will fail on the `mamba create`. I think it can just be `23.06` and use the `CONDA_USERNAME` to control whether it pulls from nightly or release.